contracts/erc2o/test/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+project(erc2o_tests)
+cmake_minimum_required(VERSION 3.12)
+
+
+include(ExternalProject)
+
+ExternalProject_Add(
+   erc2o_stubs
+   SOURCE_DIR ${CMAKE_SOURCE_DIR}/stubs/
+   BINARY_DIR ${CMAKE_BINARY_DIR}/stubs
+   CMAKE_ARGS -DCMAKE_TOOLCHAIN_FILE=${CDT_ROOT}/lib/cmake/cdt/CDTWasmToolchain.cmake -DEXTERNAL_DIR=${CMAKE_SOURCE_DIR}/external
+   UPDATE_COMMAND ""
+   PATCH_COMMAND ""
+   TEST_COMMAND ""
+   INSTALL_COMMAND ""
+   BUILD_ALWAYS 1
+)
```

Comment:
As mentioned elsewhere, by restructuring where the test contracts like the stub_evm_runtime is located, there would be no need to have any more than one ExternalProject in this entire repository.

Reply:
Refactored folder structure, this should have been resolved.